crypto/ec/ecp_secp384r1.c

```diff
@@ -0,0 +1,15385 @@
+
+#ifdef __SIZEOF_INT128__
```

Comment:
Can we alter the template so that the output looks like:

~~~C
#if defined(__SIZEOF_INT128__) && !defined(PEDANTIC)

/* bulk of 128 bit implementation */

#else

/* bulk of 32-bit implementation */

#endif
~~~

Reply:
Do you have any numbers of how this implementation performs on 64-bit platforms without `__SIZEOF_INT128__`?

Reply:
> Can we alter the template so that the output looks like:
> 
> ```c
> #if defined(__SIZEOF_INT128__) && !defined(PEDANTIC)
> 
> /* bulk of 128 bit implementation */
> 
> #else
> 
> /* bulk of 32-bit implementation */
> 
> #endif
> ```

I can do that now, I think. We made some progress in the tooling so we always have both chunks. (It's not obvious looking at this one output, but we have like 30 curves right now and were struggling to get 32-bit versions for some of them so `#else` wasn't feasible. But I think it's solved now.)

Reply:
> Do you have any numbers of how this implementation performs on 64-bit platforms without `__SIZEOF_INT128__`?

We don't because there's no research value in that :)

Reply:
> Can we alter the template so that the output looks like:
> 
> ```c
> #if defined(__SIZEOF_INT128__) && !defined(PEDANTIC)
> 
> /* bulk of 128 bit implementation */
> 
> #else
> 
> /* bulk of 32-bit implementation */
> 
> #endif
> ```

Addressed in 7e10f3b7cbb0b5f0d2ae553af4a281da80478ccf